InventoryAdminUi/Test/Mftf/Test/ReorderButtonBundleProductDefaultStockTest.xml

```diff
@@ -97,19 +95,14 @@
 
         <actionGroup ref="SaveProductFormActionGroup" stepKey="saveProduct2"/>
 
+        <magentoCLI command="indexer:reindex" stepKey="reindex"/>
+        <magentoCLI command="cache:flush" stepKey="flushCache"/>
```

Comment:
Reindex and flushing caches is bad practice and should be avoided.
You need to repeat exactly the same action that is happening to production environment.
It is usually just `cron:run`, I recommend you to use `<magentoCron stepKey="runCronIndex" groups="index"/>`